src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.LinkedList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer21 = new Bulldozer("Caterpillar 21", "Yellow", 3000, 1000);
+        Bulldozer bulldozer45 = new Bulldozer("Caterpillar 45", "Yellow", 4000, 1500);
```

Comment:
```suggestion
        Bulldozer firstBulldozer = new Bulldozer("Caterpillar 21", "Yellow", 3000, 1000);
        Bulldozer secondBulldozer = new Bulldozer("Caterpillar 45", "Yellow", 4000, 1500);
```
the same for other implementations of MachineProducer